app/bundles/CacheBundle/Config/config.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+return [
+    'routes'   => [
+        'main'   => [],
+        'public' => [],
+        'api'    => [],
+    ],
+    'menu'     => [],
+    'services' => [
+        'events'    => [
+            'mautic.cache.clear_cache_subscriber' => [
+                'class'     => \Mautic\CacheBundle\EventListener\CacheClearSubscriber::class,
+                'tags'      => ['kernel.cache_clearer'],
+                'arguments' => [
+                    'mautic.cache.provider',
+                    'monolog.logger.mautic',
+                ],
+            ],
+        ],
+        'forms'     => [],
+        'helpers'   => [],
+        'menus'     => [],
+        'other'     => [
+            'mautic.cache.provider'           => [
+                'class'     => \Mautic\CacheBundle\Cache\CacheProvider::class,
+                'arguments' => [
+                    'mautic.helper.core_parameters',
+                    'service_container',
+                ],
+            ],
+            'mautic.cache.adapter.filesystem' => [
+                'class'     => \Mautic\CacheBundle\Cache\Adapter\FilesystemTagAwareAdapter::class,
+                'arguments' => [
+                    '%mautic.cache_prefix%',
+                    '%mautic.cache_lifetime%',
+                ],
+                'tag'       => 'mautic.cache.adapter',
+            ],
+            'mautic.cache.adapter.memcached'  => [
+                'class'     => \Mautic\CacheBundle\Cache\Adapter\MemcachedTagAwareAdapter::class,
+                'arguments' => [
+                    '%mautic.cache_adapter_memcached%',
+                    '%mautic.cache_prefix%',
+                    '%mautic.cache_lifetime%',
+                ],
+                'tag'       => 'mautic.cache.adapter',
+            ],
+            'mautic.cache.adapter.redis'      => [
+                'class'     => \Mautic\CacheBundle\Cache\Adapter\RedisTagAwareAdapter::class,
+                'arguments' => [
+                    '%mautic.cache_adapter_redis%',
+                    '%mautic.cache_prefix%',
+                    '%mautic.cache_lifetime%',
+                ],
+                'tag'       => 'mautic.cache.adapter',
+            ],
+        ],
+        'models'    => [],
+        'validator' => [],
+    ],
+
+    'parameters' => [
+        'cache_adapter'           => 'mautic.cache.adapter.filesystem',
+        'cache_prefix'            => getenv('DB_NAME') ?: '%mautic.db_name%',
```

Comment:
@hluchas This param seem to be the problem based on the error message from Travis:
```
[RuntimeException]                                                           
  An error occurred when executing the "'cache:clear --no-warmup'" command:    
                                                                               
                                                                               
                                                                               
                                                                               
  In EnvVarProcessor.php line 145:                                             
                                                                               
                                                                               
    Parameter "mautic.db_name" found when resolving env var "MAUTIC_CACHE_PRE  
  FI                                                                           
    X" must be scalar, "NULL" given.  
```

Also, it's weird that the composer.json is adding new libraries but composer.lock has no change